hotline/species/templates/species/category_list.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <p><a href="{% url 'category_create' %}">Create a new category</a></p>
+    <h3>Categories</h3>
+    <table class="table table-condensed table-striped">
+        <thead>
+        <tr><th>Action</th>

```

Comment:
You need to indent your HTML. There is no excuse not to. When you do that, you'll notice missing closing tags.